drivers/softlayer/driver.go

```diff
@@ -305,11 +313,41 @@ func (d *Driver) Create() error {
 		for {
 			s, err := d.GetState()
 			if err != nil {
+				log.Debugf("Failed to GetState - %+v", err)
 				continue
 			}
 
 			if s == state.Running {
 				break
+			} else {
+				log.Debugf("Still waiting - state is %s...", s)
+			}
+			time.Sleep(2 * time.Second)
+		}
+	}
+
+	waitForSetupTransactions := func() {
```

Comment:
Nope. Was just trying to be consistent with the code above. I thought of refactoring and moving the three out, but wanted to minimize the changes.